src/components/Display/Display.spec.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Display from './Display';
+
+describe('Calculator', () => {
+    let wrapper;
+    beforeEach(() => wrapper = shallow(<Display displayValue={'0'}/>))
+
+    it('should render a div', () => {
```

Comment:
```suggestion
    it('renders a div', () => {
```
I think it's best rewritten to describe what it does, not what it should do.